### PR TITLE
Added serializable object to simplify custom objects serialization

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

**Comment by LukasRychtecky** [comment]
Dokumentaci mi přijde OK, jen bych se snažil psát v trpném rodě (ale to je asi detail).

File: docs/serializers.rst [hunk 1]
```diff
@@ -1,4 +1,73 @@
 .. _serializers:
 
 Serializers
-===========
\ No newline at end of file
+===========
+
+Returned data from resource is firstly serialized to python format via serializers. Serialization is performed recursively for complex data types (list, dict, tuple, django Model, django Queryset, etc.). Next step of serialization is converting python format response to the required format (JSON, XML, XLSX, etc.). For this purpose is used :ref:`converters`.
```
Comment (asgeirrr) [inline docs/serializers.rst]: For this purpose, converters are used.

File: docs/serializers.rst [hunk 2]
```diff
@@ -1,4 +1,73 @@
...
+
+There are several serializers every is used to serialize different type of data to get python format:
```
Comment (asgeirrr) [inline docs/serializers.rst]: There are several serializers -- each one is used to serialize a different data type to get the Python format.

File: docs/serializers.rst [hunk 3]
```diff
@@ -1,4 +1,73 @@
...
+
+ * ``StringSerializer`` - converts all string formats and escapes insecure HTML marks.
+ * ``DateTimeSerializer`` - serialize datetime and date data, automatically add server timezone.
+ * ``DictSerializer`` - recursively serializes dict content to the dict that contains only data in python format.
+ * ``CollectionsSerializer`` - recursively serializes list, tuple or set content to the generator that contains only data in python format.
+ * ``DecimalSerializer`` - serializes decimal data.
+ * ``RawVerboseSerializer`` - Pyston allows return data in three formats RAW, VERBOSE or BOTH. RAW value is the value that is stored inside database, VERBOSE is hummanized (human readable) value. The format is selected via ``serialization_format`` property (can be set via HTTP header X-Serialization-Format header). The ``RawVerboseSerializer`` gets via ``RawVerboseValue`` class which of this three formats should be returned.
```
Comment (asgeirrr) [inline docs/serializers.rst]: "The ``RawVerboseSerializer`` gets via ``RawVerboseValue`` class which of this three formats should be returned." se mi nejak nezda, chces tim rict "The ``RawVerboseSerializer`` determines which of these formats should be used using ``RawVerboseValue`` class"?

File: docs/serializers.rst [hunk 4]
```diff
@@ -1,4 +1,73 @@
...
+ * ``SerializableSerializer`` - this serializer is used for custom serialization formats :ref:`serializable`.
```
Comment (asgeirrr) [inline docs/serializers.rst]: Tohle jsem úplně nepochopil, ani z kódu, tak bych to tu možná vysvětlil trochu víc;)

**Pull request merged**